production/helm/loki/src/alerts.yaml

```diff
@@ -39,3 +39,14 @@ groups:
     for: 5m
     labels:
       severity: warning
+  name: 'loki_canaries_alerts',
+  rules:
+  - alert: 'LokiCanaryLatency',
```

Comment:
@dannykopping do we also have alerts that verify read and write path of Loki are working? E.g. on `missing_entries`.

Reply:
Not at present, but you can probably derive a few from Ed's _Loki Canary_ dashboard.
DM me if you don't have it

Do we have plans to bundle runbooks as part of Helm?

Reply:
> Do we have plans to bundle runbooks as part of Helm?

Not for now.

> derive a few from Ed's Loki Canary dashboard

Ok, it feels this should just be part of the mixin.

Reply:
Is there an issue I can upvote? I think runbooks would be easy to publish in the helmchart - we'd just need an nginx container and some markdown files really

Reply:
I've just created https://github.com/grafana/loki/issues/7310

Reply:
what's the status of putting runbooks in grafana these days? the helm chart provides dashboards as a configmap you can mount to a grafana instance, is something similar possible or in the works for runbooks?

Reply:
> Not at present, but you can probably derive a few from Ed's _Loki Canary_ dashboard. DM me if you don't have it

@dannykopping is this dashboard posted anywhere? Is that the one mentioned in the "Loki Canary" blog post?

Reply:
Which blogpost are you referring to @towolf?
We have [this](https://github.com/grafana/loki/blob/main/production/loki-mixin/dashboards/loki-canary-dashboard.libsonnet) as part of our jsonnet setup.